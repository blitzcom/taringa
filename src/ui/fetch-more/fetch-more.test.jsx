import renderer from 'react-test-renderer';

import FetchMore from './fetch-more';

describe('ui/fetch-more', () => {
  def('onClick', () => jest.fn());
  def('tree', () => renderer.create(<FetchMore onClick={$onClick} />));
  def('subject', () => $tree.toJSON());

  it('renders fetch more', () => {
    expect($subject).toMatchSnapshot();
  });

  it('calls on click', () => {
    $tree.root.findByType('button').props.onClick();
    expect($onClick).toHaveBeenCalled();
  });
});
