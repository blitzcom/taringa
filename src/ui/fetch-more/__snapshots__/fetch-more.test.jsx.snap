// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/fetch-more renders fetch more 1`] = `
<button
  className="wrapper"
  onClick={[MockFunction]}
>
  Fetch more
</button>
`;
